app/models/data_resources/resource_modules/media_content.rb

```diff
@@ -27,12 +27,17 @@ def convert_source_url_to_external_storage
     begin
       uri = URI.open(source_url.url)
       file = File.open(uri)
-      attachment.attach(io: file, filename: File.basename(file))
+      attachment.attach(
+        io: file,
+        filename: File.basename(URI.parse(source_url.url).path) || File.basename(file),
```

Comment:
fehlt hier jetzt nicht noch die header content Variante, wenn der Dateinamen darüber mitgegeben wird, wie aus dem anderen PR

```
# Versuche den Dateinamen aus dem Header zu extrahieren
# bsp.: Content-Disposition: attachment; filename=quot.pdf;
content_disposition = uri.meta['content-disposition']
original_filename = content_disposition&.match(/filename="(.+)"/i)&.[](1)
```

Reply:
also braucht es hier noch updates an der zeile?

Reply:
brauchen... wäre vermutlich nice to have gewesen, werde ich jetzt aber nicht mehr einbauen und testen